### ISSUE TITLE
Optimize SVGs for web

### ISSUE THREAD

**Issue by rlenders** [oxguy3/flags#6]
SVG images should be optimized for web. I'm not use if this is already done, optimization isn't mentioned in the readme.

I don't really know anything about SVG optimization but I do know SVGO is a popular tool for doing this.

Thoughts?

**Comment by oxguy3** [comment]
No this isn't done currently; never got around to it. The SVGs are currently just directly how they appeared in Wikipedia (and Wikipedia editors generally oppose hosting optimized SVGs). I'll look into SVGO when I've got time today.

**Comment by rlenders** [comment]
I could make a pull request from my fork on which I just made a commit that optimizes SVGs, however it also optimizes the flags added in #8 and #9 which have to be accepted first.

**Comment by rlenders** [comment]
Ah, I just noticed my fork still includes some flags that this repository does not have, those also have to be merged first. See #11

**Comment by oxguy3** [comment]
Hey, before you bother with the SVG optimization -- I think what might be preferable is for me to setup a build script for the public version of the repository that optimizes the SVGs and also generates the PNGs. I'm figuring out my options for hosting this somewhere better than GH Pages and probably on a new domain name.
